base/samples/Sonar.hpp

```diff
@@ -436,6 +436,44 @@ BASE_TYPES_DEPRECATED_SUPPRESS_START
             bins[i] = static_cast<float>(old.beam[i] * 1.0 / 255) * gain;
         pushBeam(bins, old.bearing);
     }
+
+    base::samples::SonarBeam convert2SonarBeam() {
```

Comment:
a more common pattern is to simply use the 'to' prefix, toSonarBeam, toSonarScan